### ISSUE TITLE
install instructions

### ISSUE THREAD

**Issue by bounceme** [DrabWeb/Azusa#1]
Hey, hoping to try this out without installing Xcode. i have `swiftc` , though i have never used it or swift

**Comment by DrabWeb** [comment]
I don't think you can compile without Xcode. Either way this is basically useless for an end user right now, and I wouldn't expect it to be usable for a while.
